com.github.wwmm.PulseEffects.json

```diff
@@ -0,0 +1,267 @@
+{
+  "app-id": "com.github.wwmm.PulseEffects",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.gfortran-62"],
+  "command": "pulseeffects",
+  "rename-desktop-file": "pulseeffects.desktop",
+  "rename-icon": "pulseeffects",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--own-name=com.github.wwmm.pulseeffects",
+    "--persist=.config/PulseEffects",
```

Comment:
The program uses the GtkFileChooserDialog so no it is not hard coded. It is however the default and pre-selected.

Knowing this, do you have another suggestion on how to handle this? Since the user can select the path, the most flexible thing would be to give compete access to the home directory, but that would be excessive to say the least.